Projects/MoveFlow/Sources/View/MovingFlowApartmentView.swift

```diff
@@ -0,0 +1,216 @@
+import Flow
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct MovingFlowApartmentView: View {
+    @StateObject var vm = MovingFlowViewModel()
+
+    var body: some View {
+        hForm {}
+            .hFormTitle(.standard, .title1, L10n.changeAddressEnterNewAddressTitle)
+            .sectionContainerStyle(.transparent)
+            .hFormAttachToBottom {
+                hSection {
+                    VStack(spacing: 16) {
+                        VStack(spacing: 4) {
+                            addressField().trackLoading(MoveFlowStore.self, action: .submitMoveIntent)
+                            postalAndSquareField().trackLoading(MoveFlowStore.self, action: .submitMoveIntent)
+                            numberOfCoinsuredField().trackLoading(MoveFlowStore.self, action: .submitMoveIntent)
+                            accessDateField().trackLoading(MoveFlowStore.self, action: .submitMoveIntent)
+                        }
+                        InfoCard(text: L10n.changeAddressCoverageInfoText, type: .info)
+                        LoadingButtonWithContent(
+                            MoveFlowStore.self,
+                            .submitMoveIntent
+                        ) {
+                            vm.continuePressed()
+                        } content: {
+                            hText(L10n.General.submit, style: .body)
+                        }
+                    }
+                }
+                .padding(.bottom, 8)
+                .padding(.top, 16)
+            }
+    }
+
+    func addressField() -> some View {
+        hFloatingTextField(
+            masking: Masking(type: .address),
+            value: $vm.address,
+            equals: $vm.type,
+            focusValue: .address,
+            error: $vm.addressError
+        )
+    }
+
+    func postalAndSquareField() -> some View {
```

Comment:
in the design we have "m2" (square meters) in the right placeholder which is always there. Can we add this?

Reply:
Should be fine now 👍